service/history/historyEngine.go

```diff
@@ -1192,6 +1192,7 @@ func (e *historyEngineImpl) DescribeWorkflowExecution(
 			} else {
 				p.LastStartedTime = ai.StartedTime
 			}
+			p.LastWorkerIdentity = ai.StartedIdentity
```

Comment:
do you want to overwrite p.LastWorkerIdentity even if it had been set earlier?  Or only in the case where it's missing?

Reply:
This sets the default one, overwrite if RetryLastWorkerIdentity is not empty which happen later at line 1206